katportalclient/client.py

```diff
@@ -30,12 +28,7 @@
 
 # Limit for sensor history queries, in order to preserve memory on katportal.
 MAX_SAMPLES_PER_HISTORY_QUERY = 1000000
-# Pick a reasonable chunk size for sample downloads.  The samples are
-# published in blocks, so many at a time.
-# 43200 = 12 hour chunks if 1 sample every second
-SAMPLE_HISTORY_CHUNK_SIZE = 43200
 # Request sample times  in milliseconds for better precision
-SAMPLE_HISTORY_REQUEST_TIME_TYPE = 'ms'
 SAMPLE_HISTORY_REQUEST_MULTIPLIER_TO_SEC = 1000.0
```

Comment:
Now unused.

Reply:
@tockards Please remove this line, and comment above.  No longer used.